secure/key-manager/src/counters.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_secure_push_metrics::{define_counters, Counter};
+use once_cell::sync::Lazy;
+use std::sync::Arc;
+
+// Metrics for the key manager. Uses the "libra_key_manager" prefix for all counters.
+define_counters![
+    "libra_key_manager",
+    (
+        full_consensus_rotations: Counter,
```

Comment:
completed_consensus_key_rotations

?

Reply:
I'd put the result and the key type as a label, so the stats would be like:

    libra_key_manager_key_rotations{key="consensus", result="success"}
    libra_key_manager_key_rotations{key="validator_network", result="fail"}

Reply:
@davidiw, thanks, done!
@mgorven, I see what you're saying but we only really support rotating consensus keys. I think I'll leave this as-is and if/when we support other key rotations, we can update these!

Reply:
I wouldn't call this premature optimisation, it's just good stat design. Unless we never expect key-manager to rotate anything else I think it's important to get this right.

Reply:
@mgorven, thanks! As discussed offline, it seems our crate doesn't support this yet.. 😢I'll add a TODO to the code to look into this when the times comes.